groovy/material/FirstDegreeMaterials.groovy

```diff
@@ -1029,7 +1029,25 @@ public class FirstDegreeMaterials {
                 .color(0x879e21)
                 .build()
 
-        // FREE IDs: 8245-8246
+        Alnico = new Material.Builder(8245, SuSyUtility.susyId('alnico'))
+                .dust().ingot().liquid(new FluidBuilder().temperature(1600))
+                .components(Aluminium * 5, Nickel * 4, Cobalt * 3, Copper, Iron * 11)
+                .color(0xb2d8ed)
+                .iconSet(METALLIC)
+                .flags(GENERATE_ROD, GENERATE_LONG_ROD, GENERATE_RING, GENERATE_PLATE, NO_UNIFICATION)
+                .blast(b -> b.temp(1600, GasTier.LOW).blastStats(480, 200))
```

Comment:
https://github.com/GregTechCEu/GregTech/blob/master/src/main/java/gregtech/integration/groovy/GroovyMaterialBuilderExpansion.java

there is an intended method to use with GS
![image](https://github.com/user-attachments/assets/c128f38b-d563-4c86-a3e2-45a96a5c2c83)